src/SQL_BAIN_OF_MY_EXISTENCE.sql

```diff
@@ -19,12 +19,12 @@ ADD CONSTRAINT PRIMARY KEY(ID);
 DROP TABLE IF EXISTS Hist_Avg;
```

Comment:
SQL file looks good. Logic seems to be correct and the query is quick and efficient.